packages/parser/test/dune

```diff
@@ -13,3 +14,11 @@
  (deps Runner.exe)
  (action
   (run %{deps})))
+
+(executable
+ (name Render_test)
```

Comment:
can you add a cram test that ensures the render is idempotent?

`Parse -> Print -> Parse and check equality`

This way we ensure no data is lost

Reply:
implemented this on render_test.re!